### ISSUE TITLE
New Mexico Museum of Natural History and Science

### ISSUE THREAD

**Issue by ccicero** [ArctosDB/new-collections#23]
Initial form submission in 2014, follow up by UNM?
[NewMexico_Museum_NatHist_and_Science_20141208.pdf](https://github.com/ArctosDB/new-collections/files/1536721/NewMexico_Museum_NatHist_and_Science_20141208.pdf)

**Comment by campmlc** [comment]
Yes, UNM will continue to follow up with this. They are still considering
options.
Mariel

**Comment by campmlc** [comment]
NMMNHS has hired new collections staff and wants to move forward with Arctos. They are waiting to hear about grant funding for this, but have reason to be optimistic. They asked about the potential to start entering records in test to get their Paleo dept personnel involved and trained while they wait on their funding. This would be a great opportunity to apply the proposed new funding model and offer them the 10,000 free records +$100 subscription to get started. 

One issue to resolve is that they have a Biocollections division with approx 10,000 mammals, birds, molluscs, etc., and a large Paleo collection of approx 100,000 fossil dinosaurs etc. Could we set up their various portals so they could choose to add free records to any portal up to 10,000 for limit for the institution?

**Comment by campmlc** [comment]
I should add that their current Biocollections curator is leaving in August, and it would be good to have the MOU signed before then.
